inst/include/child_model_simulation.hpp

```diff
@@ -91,10 +446,71 @@ void run_child_hiv_infections(int time_step,
           if (cpars.hc1_cd4_dist(hd) > 0) {
             state_next.children.hc1_hiv_pop(hd, 0, a, s) += state_next.base.p_infections(a, s) * cpars.hc1_cd4_dist(hd);
           }
-        }
+        } // end hc1DS
       }
-    }
-  }
+    } //end a
+  } // end NS
+
+  internal::run_calculate_perinatal_transmission_rate(time_step, pars, state_curr, state_next, intermediate);
+  //Perinatal transmission
+  for (int s = 0; s < ss.NS; ++s) {
+  for (int hd = 0; hd < hc_ss.hc1DS; ++hd) {
+       state_next.children.hc1_hiv_pop(hd, 0, 0, s) +=  state_next.children.hiv_births * intermediate.children.perinatal_transmission_rate * demog.births_sex_prop(s,time_step) * cpars.hc1_cd4_dist(hd);
+      }// end hc1DS
+      state_next.base.p_hiv_pop(0, s) +=  state_next.children.hiv_births * intermediate.children.perinatal_transmission_rate * demog.births_sex_prop(s,time_step);
+      state_next.base.p_infections(0, s) += state_next.children.hiv_births * intermediate.children.perinatal_transmission_rate * demog.births_sex_prop(s,time_step);
+
+  }// end NS
+
+
+   //Breastfeeding transmission
```

Comment:
Question about definitions, is breastfeeding not part of perinatal transmission? Or does perinatal just cover during pregnancy?